src/zigsay.zig

```diff
@@ -0,0 +1,225 @@
+const std = @import("std");
+const opt = @import("opt.zig");
+const stdout = &std.io.getStdOut().outStream().stream;
+const stdin = &std.io.getStdIn().inStream().stream;
+
+const VERSION = "0.0.1";
+
+const BUFSIZE: usize = 4096;
+const MAXLEN: usize = 39;
+
+const ZIGUANA =
+    \\     .    \             .            .
+    \\ .         \   _.--._       /|
+    \\        .    .'()..()`.    / /
+    \\            ( `-.__.-' )  ( (    .
+    \\   .         \        /    \ \
+    \\       .      \      /      ) )        .
+    \\            .' -.__.- `.-.-'_.'
+    \\ .        .'  /-____-\  `.-'       .
+    \\          \  /-.____.-\  /-.
+    \\           \ \`-.__.-'/ /\|\|           .
+    \\          .'  `.    .'  `.
+    \\          |/\/\|    |/\/\|
+    \\jro
```

Comment:
Just wondering, what is the jro?

Reply:
Credits to the author of the drawing, taken from [asciiart](https://www.asciiart.eu/animals/reptiles/lizards).

Reply:
perfect!